mining/src/mempool/model/frontier.rs

```diff
@@ -62,6 +72,11 @@ impl Frontier {
         let mass = key.mass;
         if self.search_tree.insert(key) {
             self.total_mass += mass;
+            // A decaying average formula. Denote ɛ = 1 - AVG_MASS_DECAY_FACTOR. A transaction inserted N slots ago has
+            // ɛ * (1 - ɛ)^N weight within the updated average. This gives some weight to the full mempool history while
+            // giving growing importance to more recent samples.
```

Comment:
When I first read the "giving growing importance" part, I had an impression that the more recent examples continue to increase in importance as time passes (as in something continues to give them importance, making it grow) which is actually the opposite of what happens.